codechain/__version__.py

```diff
@@ -0,0 +1,7 @@
+__title__ = "codechain"
+__description__ = "Python HTTP for Humans."
+__url__ = "https://codechain.io/"

```

Comment:
Github repository URL would be appropriate for a package.

Reply:
I fixed it.